### ISSUE TITLE
Error in module

### ISSUE THREAD

**Issue by saimon7777** [caxqueiroz/scadasim#2]
Good morning Mr. Queiroz, I was able to compile without leaving sqlite3 out. I installed Ubuntu and added all the needed libraries.

But, when I run scada_simulation_topology.ned it thows me the error: <!> Error in module (cCompoundModule) SCADA (id=1) during network setup: Class "TGMNetworkConfigurator" not found -- perhaps its code was not linked in, or the class wasn't registered with Register_Class(), or in the case of modules and channels, with Define_Module()/Define_Channel().

What can it be the broblem?

Thank you for the help.

Simon Windmuller.

**Comment by dmermi** [comment]
I have the exactly the same error message. Could not find any solution but I guess it has something to do with the Makefile. Did you get to the core of the Problem?

**Comment by saimon7777** [comment]
No, I haven't found the error, but I just noticed that it happens with all modules, When I delete the module  that "cannot be found" and run it give me the same error for the next module to simulate. Maybe is a configuration problem? Please let me know if you find anything.

**Comment by saimon7777** [comment]
Ok, I just tried with the tictoc example that come with Omnet, if you go to Run Configurations and for tictoc1 select opp_run instead of the tictoc's executable that comes preconfigured with it, it gives you similar errors of not found libraries. So definitely has to do with some configuration issue.

**Comment by dmermi** [comment]
The samples, the ones from Inet, seem to work fine for me.

**Comment by saimon7777** [comment]
Yes, for me too, It has something to do with the configuration for Scadasim

**Comment by saimon7777** [comment]
I have a problem also, in advanced (more) part of the configuration, if I dont call the sqlite3 amalgamation that I created as a project it gives me sqlite_closed not recognized undefined symbol, its weird that Omnet doesnt use the sqlite3 libraries installed in the system.
